Readme.md

```diff
@@ -0,0 +1,60 @@
+**Pre-requisite:**
+
+1. python2
+2. gcloud
+
+Follow these set-up before running the benchmarking script.
```

Comment:
These sound like steps that could be better described as "Edit <some-template-script> and set the values of these variables (e.g. ENVOY_BINARY) to the appropriate values."